XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1851,6 +1851,31 @@
             "TextBox"
           ]
         },
+        {
+          "UniqueId": "NumberBox",
+          "Title": "NumberBox",
+          "Subtitle": "A single-line or multi-line plain text field.",
```

Comment:
Sounds like this was copied from TextBox, doesn't seem entirely accurate for NumberBox.